src/TailwindBuilder.php

```diff
@@ -30,13 +30,18 @@ public function __construct(
     ) {
     }
 
-    public function runBuild(bool $watch): Process
-    {
+    public function runBuild(
+        bool $watch,
+        bool $minify,
+    ): Process {
```

Comment:
In the future it would nice to mark this method or this class `final` IMO.

Reply:
We could add it to the entire class too - and we could do it now.

Reply:
Done